python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -792,6 +794,17 @@ def process_graph(args):
 
     with open(args.conf_file, 'r', encoding="utf8") as json_file:
         process_confs = json.load(json_file)
+    if "version" not in process_confs:
+        process_confs["version"] = "gconstruct-v0.1"
+    if process_confs["version"].startswith("gsprocessing"):
+        logging.warning("Need to convert to GConstruct Config")
+        gsp_converter = GSProcessingConfigConverter()
+        process_confs = gsp_converter.convert_to_gconstruct(process_confs["graph"])
+    elif process_confs["version"].startswith("gconstruct"):
+        logging.info("Parsing config file as GConstruct config")
+    else:
+        logging.warning("Unrecognized configuration file version name: %s",
+                        process_confs["version"])
```

Comment:
If that happens we proceed to parse as a GConstruct config correct? Let's mention that.